python/hail/dataset.py

```diff
@@ -4675,6 +4674,8 @@ def sample_qc(self, root='sa.qc', keep_star=False):
         +---------------------------+--------+----------------------------------------------------------+
         | ``nTransversion``         | Int    | Number of transversion alternate alleles                 |
         +---------------------------+--------+----------------------------------------------------------+
+        | ``nStar``                 | Int    | Number of star (upstream deletion) alleles               |
```

Comment:
Shouldn't these be Int32, Float64, etc?

Reply:
yes they should

Reply:
This is wrong elsewhere in the docs too. I'll make a followup PR after this goes in.